CIWorld.cs

```diff
@@ -0,0 +1,267 @@
+﻿using Microsoft.Xna.Framework;
+using System.Collections.Generic;
+using System.IO;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.ModLoader.IO;
+
+namespace ChickenInvadersMod
+{
+    public class CIWorld : ModWorld
+    {
+        private static readonly int MaxWaves = 5;
+        private static Color DefaultColor = Color.MediumPurple;
+
+        public static bool ChickenInvasionActive = false;
+        public static bool DownedSuperChicken = false;
+        public static Vector2 SpawnLocation;
+
+        // The required amount of points for each wave
+        public class RequiredPoints
```

Comment:
This can be converted to an Array so it will look like this:
`RequiredPoints[0] = 25`